public/update.js

```diff
@@ -0,0 +1,55 @@
+var favicon= new Favico({ // creating a favico instance
```

Comment:
No need to commit `update.js`. Instead commit update.coffee

You can use js2.coffee to translate js to coffee.